core/src/test/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListenerIntegrationTest.java

```diff
@@ -58,7 +58,6 @@ public void testListenerHandlesDispatchErrorsGracefully() {
         .request(MediaType.APPLICATION_JSON_TYPE)
         .get();
     assertEquals(500, response.getStatus());
-    assertTrue(response.readEntity(String.class).contains("Resource Java method invocation error"));
```

Comment:
What is the reason for removing this? Too brittle?

Reply:
I think so, but I can't remember, to be honest.